doc/endpoint/pickler.md

```diff
@@ -87,9 +87,10 @@ import sttp.tapir.generic.Configuration
 given customConfiguration: Configuration = Configuration.default.withSnakeCaseMemberNames
 ```
 
-## Enums / sealed traits / coproducts
+## Sealed traits / coproducts
```

Comment:
I guess we can keep the `enum` in the title, that's where I would look for info on how to serialize enums :)